packages/core/src/types.ts

```diff
@@ -1279,30 +1279,38 @@ export type ExtractEvent<
   TEventType extends TEvent['type']
 > = TEvent extends { type: TEventType } ? TEvent : never;
 
-export interface ActorRef<TEvent extends EventObject, TEmitted = any>
-  extends Subscribable<TEmitted> {
+export interface BaseActorRef<TEvent extends EventObject, TEmitted = any> {
   send: Sender<TEvent>;
+  id?: string;
+  getSnapshot?: () => TEmitted | undefined;
+  stop?: () => void;
+  toJSON?: () => any;
+  subscribe?: Subscribable<TEmitted>['subscribe'];
```

Comment:
Shouldn't this `BaseActorRef` be only focused on the `send`? Isn't specifying more than that here "too much"? 🤔

Reply:
e4bc10e